docs/src/reference/grid.md

```diff
@@ -41,6 +41,7 @@ getcoordinates!
 Ferrite.ExclusiveTopology
 Ferrite.getneighborhood
 Ferrite.faceskeleton
+Ferrite.toglobal
```

Comment:
I just want to quickly discuss the naming, because it is ambiguous in distributed memory codes, where global index is the unique index of an entity in the process set. Since we call the element local index "vertex index", how do we call this "global index"?

cc @fredrikekre

Reply:
I think global/local will never be uniquely defined. Just another example would be the p4est concept where you have sort of "global" cells and "local" cells

Reply:
Let us discuss this one next week, whether we can come up with a clean solution.